JavaScript/sort/selectionSort.js

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
We already have PR under review for selection sort in Js. #2091  Please wait for us to close or merge that PR.

Reply:
But I created the issue and someone else just came and made a Pr , should it be like that? Is there any meaning in assigning anyone any issue?